crates/electrum/src/bdk_electrum_client.rs

```diff
@@ -177,9 +183,14 @@ impl<E: ElectrumApi> BdkElectrumClient<E> {
     /// - `request`: struct with data required to perform a spk-based blockchain client sync,
     ///              see [`SyncRequest`]
     /// - `batch_size`: specifies the max number of script pubkeys to request for in a single batch
-    ///              request
-    /// - `fetch_prev_txouts`: specifies whether or not we want previous `TxOut`s for fee
-    ///              calculation
+    ///                 request
+    /// - `fetch_prev_txouts`: specifies whether we want previous `TxOut`s for fee calculation. Note
+    //                         that this requires additional calls to the Electrum server, but is
+    //                         necessary for calculating the fee on a transaction if your wallet
+    //                         does not own the inputs. Methods like Wallet.calculate_fee and
+    //                         Wallet.calculate_fee_rate will return a
+    //                         CalculateFeeError::MissingTxOut error if those TxOuts are not present
+    //                         in the transaction graph.
```

Comment:
This reads well but maybe a little verbose. also lines seem to be missing an extra forward slash `/`

```rust
/// - `fetch_prev_txouts`: specifies whether we want previous `TxOut`s for fee calculation. Note
///                        that this requires additional calls to the Electrum server, but is
///                        necessary for calculating the fee on a transaction if your wallet
///                        does not own the inputs. Methods like [`TxGraph::calculate_fee`]
///                        will return an error if those `TxOut`s are missing from the
///                        transaction graph.
```

Reply:
Verbose is my middle name 😆; but people know what I mean when I'm done.

Reply:
Still it would help to have backticks and/or rustdoc links for any types that can be resolved such as `CalculateFeeError::MissingTxOut`

Reply:
Sounds good. I added the links for the error as well as the Wallet::calculate_fee and Wallet::calculate_fee_rate methods.